npm/ux/qsharp-ux.css

```diff
@@ -27,7 +50,16 @@ html {
   padding: 0;
 }
 
-body {
+html {
+  box-sizing: border-box;
+  font-size: 16px;
+}
+
+/* TODO: Make the below playground specific classes */
```

Comment:
@billti Are you planning to address this here or in a future PR?

Reply:
In a later PR. I'll probably move the styles specific to the playground out of the common package and back to a playground-specific CSS file.